Jenkinsfiles/maint-mode.yml

```diff
@@ -0,0 +1,3 @@
+pipeline:
```

Comment:
@safwanrahman Why do you think it should not be in the master branch? Is it because it only applies to the "maint-mode" branch and not the "master" branch? If that is the case, we already have other files in that category (see https://github.com/mozilla/kuma/blob/master/Jenkinsfiles/stage-integration-tests.yml for example). We need to keep it in the master branch because it allows any branch based on master to be pushed to the "maint-mode" branch and deployed as an AWS demo instance.